app/controllers/api/base_controller/parameters.rb

```diff
@@ -1,6 +1,8 @@
 module Api
   class BaseController
     module Parameters
+      include CompressedIds
```

Comment:
Wondering if moving `CompressedIds` to the top of the includes in the `BaseController` removes the need to include it (again) here? (Which I believe might also be the solution to the weird problems Jillian was running into)